selftests/runners/test_lisa_runner.py

```diff
@@ -238,8 +238,8 @@ def test_fit_a_predefined_env(self) -> None:
 
         self.verify_test_results(
             expected_test_order=["mock_ut1", "mock_ut2", "mock_ut3"],
-            expected_envs=["", "customized_0", "customized_0"],
-            expected_status=[TestStatus.SKIPPED, TestStatus.PASSED, TestStatus.PASSED],
+            expected_envs=["", "customized_0", ""],
+            expected_status=[TestStatus.SKIPPED, TestStatus.PASSED, TestStatus.SKIPPED],
```

Comment:
The code logic should be updated to support the UTs. The UT is used to verify the core logic. Every change should be thought carefully.

Reply:
hmm, I am looking into it, now the env will be marked as Bad, since it is not real env which can be ssh.

Reply:
use the `is_unittest` to skip connection test. There are some examples in code.